certvalidator/ocsp_client.py

```diff
@@ -9,10 +9,10 @@
 from . import errors
 from ._types import str_cls, type_name
 from .version import __version__
+from ._urllib import _get_host, _add_header
 
 if sys.version_info < (3,):
```

Comment:
Can we move this `if`/`else` block into `_urllib.py` and import `URLError`, `Request`, `urlopen` from `._urllib` as appropriate in `oscp_client.py` and `crl_client.py`?

I suppose an ideal further change would be to wrap `Request` on Python 2 to automatically call `util.iri_to_uri()`.

With those changes, I believe both clients should be free from know details about what version of Python they are being run on.